Readme.md

```diff
@@ -55,7 +55,7 @@ toolkit that doesn't require them.
 
 ### Ubuntu 20.04
 
-    sudo apt install python-is-python3 python3-venv libssl-dev flex bison
+    sudo apt install python-is-python3 python3-venv libssl-dev flex bison libffi-dev
```

Comment:
Why do we need this?

Reply:
This library is missing when we try to install dx-toolkit on our latest cloud_workstation with ubuntu 20.04.

When I previously created this readme,  I used slightly different environment that already had this library.

Reply:
Got it, thanks